plugins/woocommerce/client/legacy/css/_common.scss

```diff
@@ -0,0 +1,8 @@
+/**
+ * Contains rules common to all supported frontend themes.
+ */
+
+/* We do not wish to display the quantity selector (within single product pages) if the product is sold individually. */
+.woocommerce.single-product .product.sold-individually input[name="quantity"] {
+	display: none;
+}
```

Comment:
@barryhughes @Konamiman

Lots of themes add plus & minus buttons inside the `div.quantity`, which will still be displayed.

The quantity also used to be removed on the cart page, but it is still displayed there.

Reply:
Thanks for flagging that, @JimmyAppelt. Just to be clear, you're thinking it would be better to drop `input[name="quantity"]` and replace it with `.quantity` (to hide everything in that div and not just the input)?

Reply:
> The quantity used to be removed on the cart page before, but it is still displayed there.

Yep, I deliberately did not address that here. Do you see the presence of the quantity in the cart as problematic, though?

Reply:
[barryhughes](https://github.com/barryhughes) thanks for coming back to me

> Thanks for flagging that, @JimmyAppelt. Just to be clear, you're thinking it would be better to drop `input[name="quantity"]` and replace it with `.quantity` (to hide everything in that div and not just the input)?

I believe so yes 🙂 

> Yep, I deliberately did not address that here. Do you see the presence of the quantity in the cart as problematic, though?

It's not problematic 🙂, it might just be a bit strange to remove it on the single product page and leave it visible on the cart.  Showing the qty on the single product page on itself is not so problematic either.

Maybe there is a possible solution to also output the class `sold-individually` there?

Reply:
Thanks! I'll look into a further modification.

Regarding the cart quantity selector, I take your point—but I think there is also a different perspective in which this makes sense ... it really depends on the mix of products and how they are sold. However, I'll solicit some further opinions. We can certainly also solve this aspect if needed.

Reply:
I'm using the `woocommerce_before_quantity_input_field` hook and the `woocommerce_after_quantity_input_field` to add quantity increments buttons. But I can't find a way to hide my additional buttons on the cart page for sold individually products. Having the `sold-individually` class on the product row in the cart table would be a great help.

Reply:
I think we'd be open to adding that, @rajeevkarora: please consider opening a [new enhancement request](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=type%3A+enhancement%2Cstatus%3A+awaiting+triage&template=2-enhancement.yml&title=%5BEnhancement%5D%3A+), since this pull request has now been closed.

In the meantime, perhaps you could add the class you need yourself, without any template overrides, by taking advantage of the `woocommerce_cart_item_class` hook?

Reply:
Thanks @barryhughes the woocommerce_cart_item_class hook worked for me. I've never done an enhancement request before, so I'll look into doing that, thank you!